modal/runner/shared/quantize.py

```diff
@@ -0,0 +1,157 @@
+import modal.gpu
+from modal import Secret
+
+from runner.shared.common import stub
+from shared.images import BASE_IMAGE
+from shared.logging import get_logger, get_observability_secrets
+from shared.volumes import (
+    does_model_exist,
+    get_model_path,
+    models_path,
+    models_volume,
+)
+
+logger = get_logger(__name__)
+
+
+quantizer_image = (
+    BASE_IMAGE.apt_install("git")
+    .pip_install("auto-gptq==0.7.1")
+    # Use the barebones hf-transfer package for maximum download speeds. No progress bar, but expect 700MB/s.
+    .pip_install("huggingface_hub==0.20.2")
+    .pip_install("hf-transfer==0.1.4")
+    .pip_install("transformers==4.31.0")
```

Comment:
nit: could it work with the latest transformers? https://github.com/huggingface/transformers/releases